### PR TITLE
Viewer for OpenRR Nav

### PR DESCRIPTION
**Pull request by kaaatsu32329**

### CODE REVIEW

File: openrr-nav/src/dwa_planner.rs [hunk 1]
```diff
@@ -143,22 +143,30 @@ impl DwaPlanner {
         poses
     }
 
-    /// Plan the path using forward simulation
-    pub fn plan_local_path(
+    /// Get predicted plan candidates
+    pub fn predicted_plan_candidate(
```
Comment (OTL) [inline openrr-nav/src/dwa_planner.rs:147]:
```suggestion
    pub fn predicted_plan_candidates(
```

**Review by OTL — COMMENTED** [review]
Could you check some points about what I commented?

**Review by OTL — APPROVED** [review]
LGTM!

**Review by OTL — APPROVED** [review]
LGTM!

File: .github/workflows/rust.yml [hunk 2]
```diff
@@ -16,6 +16,7 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
+    - run: sudo apt-get install g++ pkg-config libx11-dev libasound2-dev libudev-dev
```
[the remark this answers is not in the document]
Comment (kaaatsu32329) [reply .github/workflows/rust.yml:19]: I tried to remove them and run ci, but all of these are needed to build.

**Pull request merged**